core/domain/exp_services.py

```diff
@@ -1793,3 +1797,188 @@ def save_multi_exploration_math_rich_text_info_model(
+                new_state_dict['interaction']['customization_args']))
+    if old_state_dict['content'] != new_state_dict['content']:
+        property_name_to_new_value_list.append(
+            (exp_domain.STATE_PROPERTY_CONTENT, new_state_dict['content']))
+
+    if old_state_dict['written_translations'] != (
+            new_state_dict['written_translations']):
+        property_name_to_new_value_list.append(
+            (
+                exp_domain.STATE_PROPERTY_WRITTEN_TRANSLATIONS,
+                new_state_dict['written_translations']))
+    if old_state_dict['interaction']['default_outcome'] != (
+            new_state_dict['interaction']['default_outcome']):
+        property_name_to_new_value_list.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_DEFAULT_OUTCOME,
+                new_state_dict['interaction']['default_outcome']))
+    if old_state_dict['interaction']['hints'] != (
+            new_state_dict['interaction']['hints']):
+        property_name_to_new_value_list.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_HINTS,
+                new_state_dict['interaction']['hints']))
+    if old_state_dict['interaction']['solution'] != (
+            new_state_dict['interaction']['solution']):
+        property_name_to_new_value_list.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION,
+                new_state_dict['interaction']['solution']))
+    if old_state_dict['interaction']['answer_groups'] != (
+            new_state_dict['interaction']['answer_groups']):
+        property_name_to_new_value_list.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_ANSWER_GROUPS,
+                new_state_dict['interaction']['answer_groups']))
+
+    for property_name, new_value in property_name_to_new_value_list:
+        change_list.append(
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': state_name,
+                'property_name': property_name,
+                'new_value': new_value
+            }))
+    return change_list
+
+
+def get_batch_of_exps_for_latex_svg_generation():
+    """Returns a batch of LaTeX strings from explorations which have LaTeX
+    strings without SVGs.
+
+    TODO(#10045): Remove this function once all the math-rich text components in
+    explorations have a valid math SVG stored in the datastore.
+
+    Returns:
+        dict(str, list(str)). The dict having each key as an exp_id and value
+        as a list of LaTeX string. Each list has all the LaTeX strings in that
+        particular exploration ID.
+    """
+
+    latex_strings_mapping = {}
+    exploration_math_rich_text_info_models = (
+        exp_models.ExplorationMathRichTextInfoModel.get_all().filter(
+            exp_models. # pylint: disable=singleton-comparison
+            ExplorationMathRichTextInfoModel.
```

Comment:
Probably this can go on previous line, for readability. (Lines with pylint comments can extend beyond the 80-char limit, I think.)

Reply:
Moving it to the previous line doesn't disable the lint-check.(that's why I kept it here)